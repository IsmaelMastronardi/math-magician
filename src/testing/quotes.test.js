import { render } from '@testing-library/react';
import ComputerQuotes from '../components/quotes';

describe('Renders a random computer quote', () => {
  test('render', () => {
    const { quote } = render(<ComputerQuotes />);
    expect(quote).toMatchSnapshot();
  });
});
