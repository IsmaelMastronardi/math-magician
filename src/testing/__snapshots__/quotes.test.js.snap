// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a random computer quote render 1`] = `undefined`;

exports[`Renders a random computer quote render 2`] = `undefined`;

exports[`Renders a result bar render 1`] = `undefined`;
